core/modules/node/node.theme.inc

```diff
@@ -294,6 +294,9 @@ function template_preprocess_node(&$variables) {
     $variables['node_url'] = FALSE;
   }
 
+  $variables['theme_hook_suggestions'][] = 'node__' . $node->type;
+  $variables['theme_hook_suggestions'][] = 'node__' . $node->type . '__' . $variables['view_mode'];
+
   // Add an extra theme hook suggestion for node id.
   $variables['theme_hook_suggestions'][] = 'node__' . $node->nid;
```

Comment:
I'd like to suggest also adding the suggestion:

```php
  $variables['theme_hook_suggestions'][] = 'node__' . $node->nid . '__' . $variables['view_mode'];

```

Putting it at the bottom will assure that it will take precedence over the others. 

I also suggest modifying the comment above this line, if the extra suggestion is added.

Reply:
Done